Dockerfile.assisted-service

```diff
@@ -1,16 +1,30 @@
-ARG BASE_TAG=stream9
+ARG RHEL_VERSION=9
+
+FROM registry.access.redhat.com/ubi$RHEL_VERSION/go-toolset:1.20 AS golang
+
+ADD . /app
+WORKDIR /app
+
+# Licenses
+RUN go install github.com/google/go-licenses@v1.6.0
+RUN ${HOME}/go/bin/go-licenses save --save_path /tmp/licenses ./...
 
 # Build binaries
-FROM quay.io/centos/centos:$BASE_TAG as builder
```

Comment:
I think this build-arg is used in another repo..
cc @carbonin

Reply:
Looks like this was already updated in openshift/release https://github.com/openshift/release/blob/9d8442afb6e832c1c722bccaef20f7c0950d227b/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L131-L134

But please to remember to consider the downstream impacts of this (if any)